### PR TITLE
Benchmarks tsne rfr

### PR DESCRIPTION
**Pull request by itearsl**

### CODE REVIEW

**Comment by Pahandrovich** [comment]
Can you rebase your branch?

**Review by KalyanovD — APPROVED** [review]

**Comment by KalyanovD** [comment]
Approved for merge after pep8 check and linux_sklearn steps fixes

File: cuml_bench/tsne.py
Comment (KalyanovD) [inline cuml_bench/tsne.py:31]: need to remove, or make it meaningful, like "TODO: investigate how to compare skl and cuml benchs"

**Comment by outoftardis** [comment]
I'll have comments for the wording of argument descriptions, please wait for my review before merging

File: cuml_bench/tsne.py
Comment (outoftardis) [inline cuml_bench/tsne.py:20]:
```suggestion
                    'the optimization is stopped.')
```

File: datasets/load_datasets.py
Comment (outoftardis) [inline datasets/load_datasets.py:88]:
```suggestion
                    'dataset(s) that should be downloaded')
```

File: datasets/load_datasets.py
Comment (outoftardis) [inline datasets/load_datasets.py:90]:
```suggestion
                        const=True, help='The list of available datasets')
```

File: datasets/load_datasets.py
Comment (outoftardis) [inline datasets/load_datasets.py:92]:
```suggestion
                        help='The datasets that should be downloaded.')
```

File: datasets/load_datasets.py
Comment (outoftardis) [inline datasets/load_datasets.py:107]:
```suggestion
            'Warning: Enumerate dataset(s) that should be downloaded')
```

File: sklearn_bench/tsne.py
Comment (outoftardis) [inline sklearn_bench/tsne.py:54]:
```suggestion
                        help='The dimension of the embedded space.')
```

File: sklearn_bench/tsne.py
Comment (outoftardis) [inline sklearn_bench/tsne.py:58]: same comment for this description as in `cuml_bench/tsne.py`

File: sklearn_bench/tsne.py
Comment (outoftardis) [inline sklearn_bench/tsne.py:63]: same comment for this description as in `cuml_bench/tsne.py`

File: sklearn_bench/tsne.py
Comment (outoftardis) [inline sklearn_bench/tsne.py:66]:
```suggestion
                        'the optimization is stopped.')
```

File: cuml_bench/tsne.py
Comment (Pahandrovich) [reply cuml_bench/tsne.py:32]:
```suggestion
# TODO: investigate how to compare sklearn and cuml metrics for tsne
```

**Review by outoftardis — APPROVED** [review]
lgtm

**Review by Pahandrovich — APPROVED** [review]

**Pull request merged**
